.github/FUNDING.yml

```diff
@@ -1,6 +1,6 @@
 # These are supported funding model platforms
 
-github: [coolacid]
+github: [ostefano]
```

Comment:
Shouldn't we remove the whole file?

Reply:
And destroy my dreams to become wealthy?

ACK, will remove.

Reply:
"OnlyFans" only as funding :joy: